pyproject.toml

```diff
@@ -13,9 +13,10 @@ build-backend = "setuptools.build_meta"
 profile = "black"
 
 [tool.cibuildwheel]
+environment-pass = ["QISKIT_PARALLEL"]
 manylinux-x86_64-image = "manylinux2014"
 manylinux-i686-image = "manylinux2014"
-test-requires = ["qiskit-terra>=0.19"]
+test-requires = ["qiskit-terra>=0.22"]
```

Comment:
Is this needed, shouldn't terra get installed with toqm because it's in the requirements list? Or am I missing something here?

Reply:
Hmm, I'm not really sure. Did you add this, or was this ported from something I had a while back?

https://github.com/qiskit-toqm/qiskit-toqm/pull/2/commits/cdb165f840896253f52c1ceb8dd7f88e129747c5#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711R18

Reply:
Oh, it looks like you simply moved it from where I had it in the CI YAML.